package.json

```diff
@@ -68,6 +68,5 @@
     "ie 11"
   ],
   "dependencies": {
```

Comment:
it'd probably be cleaner to remove this than leave an empty `dependencies` block (I noticed the same thing in the earlier PR as well)

Reply:
true, npm usually removes it for you if you remove it via `npm uninstall` i think